app/routes/forgot_password.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
file names `forgot_password.tsx`, `otp_verification` and `reset_password` should be in `UpperCamelCase` following the standard practice.

Reply:
here file name is following the route covention.

Reply:
Could you share the reference in the remix docs please
https://remix.run/docs/en/v1/guides/routing

Reply:
> 

https://remix.run/docs/en/v1/guides/routing#pathless-layout-routes
I read this and that's why the names

Reply:
> > 
> 
> https://remix.run/docs/en/v1/guides/routing#pathless-layout-routes I read this and that's why the names

That is a different topic, not file names. You can use two underscores at the start of a folder or file name inside routes to make it a `pathless layout route`. They have given an example for the same there:
- if you have `login.jsx` inside `auth` folder, your url path will show `/auth/login` but you wouldn't want `/auth/` in every such path and to handle that you can put two underscores at the start of that folder so the url path will then show `/login` directly.

While here we have `snake_case` in the file names.

Reply:
snake_case doesn't create pathless-layout-routes and We could go ahead with the routes now, when Integration and demo, If we observe the route is getting to big, we could decide at that time

Reply:
addressed the comment @harshith-venkatesh